docs/public/static/diffs/template-bare-minimum/raw/46..47.diff

```diff
@@ -0,0 +1,535 @@
+diff --git a/templates/expo-template-bare-minimum/android/app/build.gradle b/templates/expo-template-bare-minimum/android/app/build.gradle
```

Comment:
Are these changes related to the upgrade?

Reply:
we could skip generating the diff for each rc bump. just to do it before cuting sdk 51 branch or before sdk 51 release.